Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -98,6 +98,9 @@
         type: InstrumentBoundUserInterface
       enum.HealthAnalyzerUiKey.Key:
         type: HealthAnalyzerBoundUserInterface
+      # RMC-14: Add RMC health scanner UI key. (I'm just copying upstream okay... I know this is out of place. :( )
+      enum.HealthScannerUIKey.Key:
+        type: HealthScannerBoundUserInterface
```

Comment:
if you move this entire component onto the RMC pda you dont have to touch this file